README.md

```diff
@@ -22,7 +22,8 @@
 
 - A **Raspberry Pi** including a correctly assembled **7" touch display v1.1 or higher** running a Linux-based OS
 - Python 3.5 or higher
-- Optional: `pygobject` for the GUI, is likely to be already installed on a recent Raspbian
+- For Raspberry Pi: Optional: `pygobject` for the GUI, is likely to be already installed on a recent Raspbian
+- For Tinkerboard: ``gir1.2-gtk-3.0`` for the GUI, install via sudo apt-get install gir1.2-gtk-3.0
```

Comment:
Let's make it more clear that the GUI in general is optional, not only on the RPi:

```markdown
- For the GUI (optional):
  - For Raspberry Pi: `pygobject`, is likely to be already installed on a recent Raspbian Desktop
  - For Tinkerboard: ``gir1.2-gtk-3.0``, install via `sudo apt-get install gir1.2-gtk-3.0`
```

- For the GUI (optional):
  - For Raspberry Pi: `pygobject`, is likely to be already installed on a recent Raspbian Desktop
  - For Tinkerboard: ``gir1.2-gtk-3.0``, install via `sudo apt-get install gir1.2-gtk-3.0`

Reply:
I wasn't able to python install without gir so I didn't put that as optional

Reply:
fixed it anyway